endless-sky.6

```diff
@@ -38,6 +38,9 @@ sets the directory where preferences and saved games will be stored.
 .IP \fB\-p,\ \-\-parse\-save
 prints any content or whitespace\-formatting errors found while loading data files and the most recent saved game. This option prevents the game from launching.
 
+.IP \fB\-\-parse\-images
```

Comment:
I like the suggestion from mOctave to use `--parse-assets`, since it is more than just images and sounds. If we have music, then that is included, and images can be different things like ship models, landscape images, parts of the UI.

Also, this parameter says parse-images, while the ci.yml has a longer name, so it needs attention in some way.